.docker/entrypoints/app.sh

```diff
@@ -10,7 +10,7 @@ fi
 
 if [ "$RAILS_ENV" == "test" ]; then
   bundle check || bundle install
-  rails db:setup RAILS_ENV=$RAILS_ENV
+  rails db:create db:schema:load RAILS_ENV=$RAILS_ENV
```

Comment:
`rails db:setup` will also seed the database: https://edgeguides.rubyonrails.org/active_record_migrations.html#setup-the-database. This leads to the fact, that we always already have some `id`s. Therefore the sequence does not work.